### ISSUE TITLE
Ask for the real size and md5sum for chunks listed in openio object locate command in erasurecode policy

### ISSUE THREAD

**Issue by sebastienlapierre** [open-io/oio-sds#1066]
In erasurecode policy, we really need having the real size and md5sum for chunks listed in `openio object locate` command and not the size and md5sum of the content.
**Example:** 
content: 
```
+-----------+----------------------------------+
| Field     | Value                            |
+-----------+----------------------------------+
| account   | openio_account                   |
| container | mycontainer                      |
| ctime     | 1498773704                       |
| hash      | 272913026300E7AE9B5E2D51F138E674 |
| id        | 7D09457220530500F512FF3F038865AB |
| mime-type | application/octet-stream         |
| object    | myobject                         |
| policy    | EC14_4_liberasure                |
| size      | 111                              |
| version   | 1498773704739199                 |
+-----------+----------------------------------+
```

locate:
```
+------+-------------------------------------------------------------------------------------------+------+----------------------------------+
| Pos  | Id                                                                                        | Size | Hash                             |
+------+-------------------------------------------------------------------------------------------+------+----------------------------------+
| 0.0  | http://172.16.139.3:6040/494F41213C25D33F1EB457864334AA5E0F155730DEA99CB1321E44962D591B8B |  111 | 272913026300E7AE9B5E2D51F138E674 |
| 0.1  | http://172.16.139.1:6038/328F989DAA9E5487269E63A510309A993EF47AD6DFDAD6877238B4B3427C143F |  111 | 272913026300E7AE9B5E2D51F138E674 |
| 0.10 | http://172.16.139.3:6046/83DA4A422AD158E9E8B5C49F45A7692C7EC7E9D35462100A27D1AB1A9CC83A5B |  111 | 272913026300E7AE9B5E2D51F138E674 |
| 0.11 | http://172.16.139.2:6037/E9215BFE921F7572E8AED929CE30D1044CD425F2F1FA191765ADA1031D33CA77 |  111 | 272913026300E7AE9B5E2D51F138E674 |
| 0.12 | http://172.16.139.2:6050/68CF2CA4D5131F7E12FB7537D9BBA09E451F28E6B88D581600938C62B73473B1 |  111 | 272913026300E7AE9B5E2D51F138E674 |
| 0.13 | http://172.16.139.1:6031/C5B88BE07577D0D86374C89F6654083BCE95B6FF35E5728614F95E993C06E47B |  111 | 272913026300E7AE9B5E2D51F138E674 |
| 0.14 | http://172.16.139.2:6030/4616F3DE6B540EBBF185FB7D42CFC73840AD9D2C0EDE8673D28EDDD3D6B46A9A |  111 | 272913026300E7AE9B5E2D51F138E674 |
| 0.15 | http://172.16.139.3:6033/05340F5E8004C58ACCA6FA4D72392DC34471EEF81A24F27292E48323F38273A0 |  111 | 272913026300E7AE9B5E2D51F138E674 |
| 0.16 | http://172.16.139.1:6027/0E99547C16C7354A61FCCA2CE067083E4DB79DEE14E038A6A6A735EC37097E6B |  111 | 272913026300E7AE9B5E2D51F138E674 |
| 0.17 | http://172.16.139.1:6045/50A1C610FF4E3F4F8CA585CA8860EE4E79F2E349EE2D06B3A6D4D34710B5F533 |  111 | 272913026300E7AE9B5E2D51F138E674 |
| 0.2  | http://172.16.139.2:6044/A54FAE9D28031F2BCAF9C61AF0BC76A0DF93F65F55D01335CCD619E150AFF5D0 |  111 | 272913026300E7AE9B5E2D51F138E674 |
| 0.3  | http://172.16.139.3:6027/A12436D6D5F912E246E2054EEF5D0D4C059CE5413BCE6A871E7A186313EA522B |  111 | 272913026300E7AE9B5E2D51F138E674 |
| 0.4  | http://172.16.139.2:6025/D4FA3D8EB179B9C71891B105B778858C13FB1031C1BD8B07139855679782F1D6 |  111 | 272913026300E7AE9B5E2D51F138E674 |
| 0.5  | http://172.16.139.2:6035/20519099B2A11D99C8DA5815BCA1D4745A26C527539707FF7C39E03462BAB36B |  111 | 272913026300E7AE9B5E2D51F138E674 |
| 0.6  | http://172.16.139.1:6030/66EAA6DEFC2BE900FE6CB6D504C63B8F09A2A657D4211539A7663E944ECF9C20 |  111 | 272913026300E7AE9B5E2D51F138E674 |
| 0.7  | http://172.16.139.3:6029/8319D82FF81A2D08B96C540CA7E80A6F0020C3693AB25824E2A6A29AA1492003 |  111 | 272913026300E7AE9B5E2D51F138E674 |
| 0.8  | http://172.16.139.3:6047/5359529DA0E7B5E84D8552BBFB2917B2C27B42054998EC0D626053FBC74F049E |  111 | 272913026300E7AE9B5E2D51F138E674 |
| 0.9  | http://172.16.139.1:6034/C33044BE684B0975F63258A10F166CE985E13D47CD6195D0BC8499CE1B817941 |  111 | 272913026300E7AE9B5E2D51F138E674 |
+------+-------------------------------------------------------------------------------------------+------+----------------------------------+
```

**Comment by fvennetier** [comment]
The `Size` and `Hash` columns represent the size and checksum of the **metachunk**. We can make it clearer if you want, by renaming the columns headers to `Metachunk size` and `Metachunk Hash`.

Displaying the size and checksum of the chunks as they are on rawx services is a bit more difficult, since this information is not stored in meta2 database. To get them, we need to send `HEAD` request to each chunk, which will take some time. We can do that, but as an extra option, the default will be to display only the information known by meta2.

**Comment by sebastienlapierre** [comment]
I agree for an extra option in HEAD request, it may be very usefull for check or test purposes.

**Comment by fvennetier** [comment]
Implemented in #1077.
